package.json

```diff
@@ -86,7 +86,7 @@
     "morgan": "1.2.0",
     "ms": "0.7.1",
     "node-sass": "3.7.0",
-    "notifications-panel": "1.1.11",
+    "notifications-panel": "automattic/notifications-panel#add\/104-settings-button",
```

Comment:
yeah @roundhill I just didn't.

```bash
rm -rf node_modules/notifications-panel
npm install 'automattic/notifications-panel#add/104-settings-button'
```

but now I will shrinkwrap (didn't earlier because make was running lol)